packages/blockchain/src/airdrop/deployThirdwebAirdropContract.ts

```diff
@@ -16,16 +18,16 @@ export async function deployThirdwebAirdropContract({
   totalAirdropAmount,
   expirationTimestamp,
   openClaimLimitPerWallet,
-  trustedForwarders,
-  proxyFactoryAddress,
-  implementationAddress,
+  trustedForwarders = [],
+  proxyFactoryAddress = THIRDWEB_AIRDROP_PROXY_FACTORY_ADDRESS,
```

Comment:
i moved these constants down in the stack since it was just being imported in multiple places